server/routes/ticket.py

```
@@ -302,5 +300,333 @@ def post(self):
+                "resolved": resolved_count,
+                "closed": closed_count,
+            }
+
+            return restful_response(
+                status="success",
+                message="Ticket stats fetched successfully",
+                data=stats,
+                status_code=200,
+            )
+
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)